checkstyle/suppressions.xml

```diff
@@ -40,6 +40,7 @@
               files="core[\\/]src[\\/](generated|generated-test)[\\/].+.java$"/>
     <suppress checks="NPathComplexity" files="(ClusterTestExtensions|KafkaApisBuilder|SharePartition).java"/>
     <suppress checks="NPathComplexity|ClassFanOutComplexity|ClassDataAbstractionCoupling" files="(RemoteLogManager|RemoteLogManagerTest).java"/>
+    <suppress checks="MethodLength" files="RemoteLogManager.java"/>
```

Comment:
For my understanding, what is causing this? I am somewhat surprised that given you are breaking down methods into smaller ones to see this popping up

Reply:
It is caused by the method _cleanupExpiredRemoteLogSegments_ which is a little longer now.

> Message: Method length is 173 lines (max allowed is 170).

We have added a few lines in this method to ensure segments that are currently being copied (but copy isn't complete yet), are not expired by the expiration task.